src/settings/global_user.rs

```diff
@@ -1,12 +1,21 @@
 use std::env;
+use std::fs;
 use std::path::{Path, PathBuf};
 
 use cloudflare::framework::auth::Credentials;
-use log::info;
+use config;
```

Comment:
i did this to be obnoxious. there's even a commit that says it. it was spiteful and i can revert 😂 but there is an issue out for removing config from the project.